Clarify behavior of `placement_group_capture_child_tasks` in docs (#46885)
ray-project/ray: doc/source/ray-core/scheduling/placement-group.rst

  :start-after: __child_capture_disable_pg_start__
  :end-before: __child_capture_disable_pg_end__

.. warning::

>   The value of ``placement_group_capture_child_tasks`` for a given actor is not inherited from its parent. If are creating nested actors of depth greater than 1 

Author: ```suggestion
  The value of ``placement_group_capture_child_tasks`` for a given actor is not inherited from its parent. If you are creating nested actors of depth greater than 1 
```

>   The value of ``placement_group_capture_child_tasks`` for a given actor is not inherited from its parent. If you are creating nested actors of depth greater than 1 

Reviewer: ```suggestion
  The value of ``placement_group_capture_child_tasks`` for a given actor isn't inherited from its parent. If you're creating nested actors of depth greater than 1 
```

>   that should all use the same placement group, ``placement_group_capture_child_tasks`` should be explicitly set for each actor.

Reviewer: ```suggestion
  and should all use the same placement group, you should explicitly set ``placement_group_capture_child_tasks`` explicitly set for each actor.
```
